frontend/src/scenes/persons/personsLogic.ts

```diff
@@ -52,6 +60,34 @@ export const personsLogic = kea<personsLogicType<PersonPaginatedResponse>>({
                 },
             },
         ],
+        person: [
+            null as PersonType | null,
+            {
+                loadPerson: async (id) => {
```

Comment:
Let's add an explicit type to `id` as well as for the return value here - otherwise ts thinks this is `any`

Reply:
I've added the return type to the function, though I'm not sure if we should do this. It was my understanding that the kea typegen magic handle this automatically based on the type set on line 64, @mariusandra ?

Reply:
Kea will automatically infer the types typescript is able to infer - in this case though it thinks it's dealing with `any` which is a unescapable black box.